schedule/yam/test_cases/yast_hostname.yaml

```diff
@@ -3,8 +3,6 @@ name: yast_hostname
 description: >
   Test suite uses default option to set hostname. Test validates if installation
   can successfully start in case of usage of these parameters.
-vars:
-  YUI_REST_API: 1
```

Comment:
You removed the setting of YUI_REST_API from yaml file, need add the setting through a MR. Other LGTM.

Reply:
Just saw you have a MR https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/296, I will review it also. Please add related PR/MR in your PR/MR to remind people to review related PR/MR.